compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -159,6 +159,15 @@ def DecomposePackUnPackOpsPass :
   ];
 }
 
+def DecomposeBoundaryPackUnPackOpsPass :
```

Comment:
Where is this new pass used?

Reply:
It will be used in https://github.com/iree-org/iree/pull/18565